### DISCUSSION TITLE
Potential conflict with kitty terminal themes?

### DISCUSSION THREAD

**Discussion by Icecreambobcat** [NvChad/NvChad#3001] [Q&A]
After temporarily toggling theme transparency in `nvim`, having kitty use gruvbox dark and `nvim` use the gruvbox theme while kitty is transparent makes `nvim` unexpectedly transparent when this theme collision doesn't occur anywhere else.

**tl;dr**
after setting transparency on and off again the background transparency can't be disabled when both kitty and nvchad have gruvbox (dark) set

**Comment by siduck** [comment]
show   image

**Comment by Icecreambobcat** [comment]
<img width="1440" alt="Screenshot 2024-09-26 at 1 10 54 am" src="https://github.com/user-attachments/assets/67df149a-1e28-45ee-a4d1-ea78231bdf49">
idk how much this will help
expected behaviour is that there is no transparency on LHS term
kitty itself is set to 0.85 transparency

**Reply by Icecreambobcat** [reply]
other themes are not affected by this at all from the ones that I've gone through

**Reply by siduck** [reply]
this is weird... toggling transparency should work, can u make a video

**Reply by Icecreambobcat** [reply]
https://github.com/user-attachments/assets/aee6afed-ffcd-48f2-a11c-488633fafc40

yeah i have no clue
tried to `rm -rf` the theme directories and tried reinstalling base46 but that did nothing

**Reply by siduck** [reply]
no idea tbh, can u try with other terminals

**Reply by Icecreambobcat** [reply]
tried with iterm
every theme just ends up transparent regardless of chadrc if transparency is set in iterm

**Reply by Icecreambobcat** [reply]
looks like its just unintentionally taking the bg colour of whatever terminal its in

**Reply by siduck** [reply]
try asking in my discord server, someone might know the issue!
